.gitignore

```diff
@@ -1,4 +1,3 @@
-.chalice
```

Comment:
I had added this to the gitignore so that I didn't accidentally stage "other" .chalice files. You can get around .gitignore using git add file -f.